app/experimenter/legacy-ui/rapid/components/experiments/ExperimentDetails.tsx

```diff
@@ -279,6 +280,13 @@ const ExperimentDetails: React.FC = () => {
           </tbody>
         </table>
 
+        {experimentData.analysis?.show_analysis && (
```

Comment:
HA! Interesting! This new `.thing?` syntax, god that's so handy

Reply:
I know!!!! @jodyheavener showed me this and I love it!

Reply:
[Coming soon](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining) to a JavaScript near you